### DISCUSSION TITLE
Removing extra whitespace between authors when displaying bibliography

### DISCUSSION THREAD

**Discussion by ghbrooks28** [alshedivat/al-folio#2153] [Q&A]
How can I remove the extra space before and after an author's full name in the bibliography?
![author_extra_space](https://github.com/alshedivat/al-folio/assets/142257247/5a63349c-94ba-4392-afba-6e5fc1fa1fc8)

**Comment by george-gca** [comment]
Basically in the code somewhere is missing a `-`. More specifically, this is due to `liquid` code creating whitespaces, and the way to fix this is by using `{%-` instead of `{%` or `-%}` instead of `%}` depending on the case. Here is an [explanation from liquid documentation](https://shopify.github.io/liquid/basics/whitespace/).

If you manage to fix this and is willing to submit a PR, please do so.

**Reply by george-gca** [reply]
Fixed in #2197
